docs/customosbuildinstructions.md

```diff
@@ -105,6 +105,12 @@ Here are the expected contents of each subdirectory /file
              /bin/iproute
              /bin/hostname
 
+    - Required binaires: utilities used by docker
+
+             /bin/ls
+             /bin/cat
+             /bin/test
```

Comment:
To validate if a directory exists when looking up a disk device from a controller/LUN. Part of the dynamic sandbox management stuff.